concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md

```diff
@@ -46,6 +46,57 @@ Regulating the empty space on a bar's sides controls the relative bar width. Use
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart ... >
+            <DxSeries :bar-padding="0.1"/> <!-- for an individual series -->
+            <DxSeries ... />
+            ...
+            <DxCommonSeriesSettings :bar-padding="0.1"/> <!-- for all series in the chart -->
+        </DxChart>
+    </template>
+
+    <script>
+    import DxChart, {
+        DxSeries,
+        DxCommonSeriesSettings
+    } from 'devextreme-vue/chart';
+
+    export default {
+        components: {
+            DxChart,
+            DxSeries,
+            DxCommonSeriesSettings
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import Chart, {
+        Series,
+        CommonSeriesSettings
+    } from 'devextreme-react/chart';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Chart ... >
+                    <Series barPadding={0.1} /> <!-- for an individual series -->
+                    <Series ... />
+                    ...
+                    <CommonSeriesSettings barPadding={0.1}/> <!-- for all series in the chart -->
```

Comment:
Please fix this in all React examples in this PR.

```suggestion
                    <CommonSeriesSettings barPadding={0.1}/> {/* for all series in the chart */}
```

Reply:
Fixed